ruby.rb

```diff
@@ -7,4 +7,32 @@
 # Add comments to your code explaining your approach
 
 class Bike
+            #attr builds attributes we need on initializtion. 
+            attr_accessor :bike, :model, :wheels, :current_speed
+            # bike model Trek has to exist
+            # bike has to have 2 wheels and a current speed of zero
+            def initialize(bike)
+                @bike = bike
```

Comment:
since Bike is the class, the bike attributes are the model, wheels, and speed. I don't think your bike instance variable is actually doing anything for you.